### PR TITLE
Bug 1840090 - Update content description string for shopping experience

### PR DESCRIPTION
**Pull request by Alexandru2909**
Update content description string for shopping experience.

### Pull Request checklist

- [x] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Changelog**: This PR includes [a changelog entry](https://github.com/mozilla-mobile/firefox-android/blob/main/docs/changelog.md) or does not need one
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/firefox-android/blob/main/docs/shared/android/accessibility_guide.md) or does not include any user facing features

### After merge
- **Breaking Changes**: If this is a breaking Android Components change, please push a draft PR on [Reference Browser](https://github.com/mozilla-mobile/reference-browser) to address the breaking issues.

### To download an APK when reviewing a PR (after all CI tasks finished running):
1. Click on `Checks` at the top of the PR page.
2. Click on the `firefoxci-taskcluster` group on the left to expand all tasks.
3. Click on the `build-apk-{fenix,focus,klar}-debug` task you're interested in.
4. Click on `View task in Taskcluster` in the new `DETAILS` section.
5. The APK links should be on the right side of the screen, named for each CPU architecture.

### GitHub Automation
https://bugzilla.mozilla.org/show_bug.cgi?id=1840090

### CODE REVIEW

File: fenix/app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -2235,7 +2235,7 @@
     <!-- Flag showing that the review quality check feature is work in progress. -->
     <string name="review_quality_check_beta_flag" tools:ignore="UnusedResources">Beta</string>
     <!-- Content description (not visible, for screen readers etc.) for opening browser menu button to open review quality check bottom sheet. -->
-    <string name="browser_menu_review_quality_check">Open Review quality check</string>
+    <string name="browser_menu_review_quality_check">Open review checker</string>
```
Comment (flodolo) [inline fenix/app/src/main/res/values/strings.xml:2238]: https://mozilla-l10n.github.io/documentation/localization/making_string_changes.html

You need a new ID. Also, what about the next string?

**Review by flodolo — CHANGES_REQUESTED** [review]
(forgot to use the review changes button)

File: fenix/app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -2235,9 +2235,13 @@
     <!-- Flag showing that the review quality check feature is work in progress. -->
     <string name="review_quality_check_beta_flag" tools:ignore="UnusedResources">Beta</string>
     <!-- Content description (not visible, for screen readers etc.) for opening browser menu button to open review quality check bottom sheet. -->
-    <string name="browser_menu_review_quality_check">Open Review quality check</string>
+    <string name="browser_menu_review_quality_check" moz:removedIn="121" tools:ignore="UnusedResources">Open Review quality check</string>
```
Comment (flodolo) [inline fenix/app/src/main/res/values/strings.xml:2238]: Since this string only landed in Nightly and it's unused, I assume you could just remove it?

File: fenix/app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -2235,9 +2235,9 @@
     <!-- Flag showing that the review quality check feature is work in progress. -->
     <string name="review_quality_check_beta_flag" tools:ignore="UnusedResources">Beta</string>
     <!-- Content description (not visible, for screen readers etc.) for opening browser menu button to open review quality check bottom sheet. -->
-    <string name="browser_menu_review_quality_check">Open Review quality check</string>
+    <string name="browser_menu_review_quality_check_1">Open review checker</string>
     <!-- Content description (not visible, for screen readers etc.) for closing browser menu button to open review quality check bottom sheet. -->
-    <string name="browser_menu_review_quality_check_close">Close review quality check</string>
+    <string name="browser_menu_review_quality_check_close_1">Close review checker</string>
```
Comment (flodolo) [inline fenix/app/src/main/res/values/strings.xml:2240]: Might want to remove the trailing spaces, otherwise someone else's editor might do it next time, and create an unnecessary diff.

**Review by flodolo — APPROVED** [review]

File: fenix/app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -2235,9 +2235,9 @@
     <!-- Flag showing that the review quality check feature is work in progress. -->
     <string name="review_quality_check_beta_flag" tools:ignore="UnusedResources">Beta</string>
     <!-- Content description (not visible, for screen readers etc.) for opening browser menu button to open review quality check bottom sheet. -->
-    <string name="browser_menu_review_quality_check">Open Review quality check</string>
+    <string name="browser_menu_review_quality_check_1">Open review checker</string>
```
Comment (gabrielluong) [inline fenix/app/src/main/res/values/strings.xml:2238]: Since the string will be the same as `review_quality_check_second_cfr_action`, I am wondering if we should've have this as a separate string. I do understand having separate strings for content description.

File: fenix/app/src/main/res/values/strings.xml [hunk 5]
```diff
@@ -2235,9 +2235,9 @@
...
     <!-- Content description (not visible, for screen readers etc.) for closing browser menu button to open review quality check bottom sheet. -->
-    <string name="browser_menu_review_quality_check_close">Close review quality check</string>
+    <string name="browser_menu_review_quality_check_close_1">Close review checker</string>
```
Comment (gabrielluong) [inline fenix/app/src/main/res/values/strings.xml:2240]: The `browser_menu_` prefix is a bit unfortunate here since these strings don't actually appear in the browser menu.

Comment (Alexandru2909) [reply fenix/app/src/main/res/values/strings.xml:2240]: Should we update the string name to "review_quality_check_close" instead?

Comment (flodolo) [reply fenix/app/src/main/res/values/strings.xml:2240]: Please don't, these strings are being translated at this point, and this would require retranslation (for the 3rd time).

File: fenix/app/src/main/res/values/strings.xml (hunk 4 above)
Comment (Alexandru2909) [reply fenix/app/src/main/res/values/strings.xml:2238]: I think it's good to have them in separate strings, in case the CFR action string changes/gets removed in the future we'll still have the content description unaffected.
